### PR TITLE
Added some methods+operators to FloatPoint2DVectorList

### PR DESCRIPTION
**Pull request by marcusnaslund**
Useful for #784

### CODE REVIEW

**Comment by marcusnaslund** [comment]
peer review @sebastianbaginski ?

File: source/geometry/FloatPoint2DVectorList.ooc [hunk 1]
```diff
@@ -118,6 +118,28 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		}
 		result
 	}
+	minValues: func -> FloatPoint2D {
+		result: FloatPoint2D
+		thisPointer := this pointer as FloatPoint2D*
```
Comment (sebastianbaginski) [inline source/geometry/FloatPoint2DVectorList.ooc]: This could be moved into the `else` block

File: source/geometry/FloatPoint2DVectorList.ooc [hunk 2]
```diff
@@ -118,6 +118,28 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
...
+			result = FloatPoint2D new()
+		else
+			result = thisPointer[0]
+			for (i in 1 .. this _count)
+				result = result minimum(thisPointer[i])
+		result
+	}
+	maxValues: func -> FloatPoint2D {
+		result: FloatPoint2D
+		thisPointer := this pointer as FloatPoint2D*
```
Comment (sebastianbaginski) [inline source/geometry/FloatPoint2DVectorList.ooc]: Same here

Comment (marcusnaslund) [inline source/geometry/FloatPoint2DVectorList.ooc]: oh wow, there are no brackets after `else`

**Pull request merged**